packages/runtime/src/runtimes/react/components/LiveProvider.tsx

```diff
@@ -1,27 +1,16 @@
-import { ReactNode, useMemo } from 'react'
+import { type PropsWithChildren, useMemo } from 'react'
 
-import { StoreContext } from '../hooks/use-store'
 import * as ReactPage from '../../../state/react-page'
-import { MakeswiftHostApiClient } from '../../../api/react'
-import { useReactRuntime } from '../../../next/context/react-runtime'
-import { MakeswiftHostApiClientProvider } from '../../../next/context/makeswift-host-api-client'
 
-type Props = {
-  client: MakeswiftHostApiClient
-  rootElements?: Map<string, ReactPage.Element>
-  children?: ReactNode
-}
+import { useReactRuntime } from '../hooks/use-react-runtime'
+import { StoreContext } from '../hooks/use-store'
 
-export default function LiveProvider({ client, children, rootElements }: Props): JSX.Element {
+export default function LiveProvider({ children }: PropsWithChildren): JSX.Element {
   const runtime = useReactRuntime()
   const store = useMemo(
-    () => ReactPage.configureStore({ preloadedState: runtime.store.getState(), rootElements }),
-    [rootElements, runtime],
```

Comment:
`rootElements` are filled in through [`useRegisterDocuments` effect](https://github.com/makeswift/makeswift/pull/836/files#diff-574436562edd329137174ba80e6ecdb186f78f8e2b7e6cbc929afe8ede60e4e9R33)